core/src/main/java/com/eternalcode/core/feature/essentials/mob/MobFilter.java

```diff
@@ -0,0 +1,7 @@
+package com.eternalcode.core.feature.essentials.mob;
+
+import org.bukkit.entity.Entity;
+
+public interface MobFilter {
```

Comment:
Jako że tutaj mamy interface tylko dla funkcji to dodaj adnotacje  `FunctionalInferface` oraz spacje pomiedzy voidem :)